samples/account-linking/csharp/Manifest/manifest.json

```diff
@@ -0,0 +1,96 @@
+{
+    "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.12/MicrosoftTeams.schema.json",
+    "version": "1.0.0",
+    "manifestVersion": "1.12",
+    "id": "19b5d434-5d4b-47a8-800f-446b6a010ba2",
+    "packageName": "com.package.name",
```

Comment:
Is this something that should be unique? If someone was testing multiple manifests with the same package name does one affect the other? How is this used by the actual client?

Reply:
I honestly have no clue what that field does. Dev portal added it. I'll remove it from this manifest since I can't justify why it is here.